core/lsp/PreemptionTaskManager.h

```diff
@@ -19,19 +19,22 @@ class PreemptionTaskManager final {
     std::shared_ptr<TypecheckEpochManager> epochManager;
     // Thread ID of the typechecking thread. Lazily set.
     std::optional<std::thread::id> typecheckingThreadId;
-    // Thread ID of the message processing thread. Lazily set.
-    std::optional<std::thread::id> messageProcessingThreadId;
+    // Thread ID of the processing thread. Lazily set.
+    std::optional<std::thread::id> processingThreadId;
```

Comment:
I renamed the "message processing thread" to "processing thread", since it now operates on tasks.